### DISCUSSION TITLE
OPDSPublication format type

### DISCUSSION THREAD

**Discussion by 838** [readium/architecture#174] [Q&A] [answered]
What will be the correct why of specifying that a OPDSPublication is a webaudiobook/webpub or PDF? 

[OPDS Publication schema](https://drafts.opds.io/schema/publication.schema.json)

**Comment by mickael-menu** [comment, marked as answer]
Take a look here:
* https://readium.org/webpub-manifest/profiles/audiobook.html#1-metadata
* https://readium.org/webpub-manifest/profiles/epub.html#1-declaring-conformance-with-the-epub-profile

But basically, you need to declare the profile conformed by your RWPM, in a [`metadata.conformsTo` property](https://readium.org/webpub-manifest/schema/metadata.schema.json). The values are:

* EPUB: https://readium.org/webpub-manifest/profiles/epub
* PDF: https://readium.org/webpub-manifest/profiles/pdf
* Audiobook: https://readium.org/webpub-manifest/profiles/audiobook
* Comic book/Divina: https://readium.org/webpub-manifest/profiles/divina

For a basic Web Publication, you don't need any profile.

**Reply by 838** [reply]
@mickael-menu 🙏

**Reply by mickael-menu** [reply]
Ha actually re-reading I have a doubt. Do you mean to specify the format of the acquisition links in your OPDS publication?

In this case you just need to provide the media type in the link's `type` property, for example:

```json
{
  "rel": "http://opds-spec.org/acquisition",
  "href": "https://catalog.feedbooks.com/item/4450593/preview",
  "type": "application/epub+zip"
}
```

See a full example here https://catalog.feedbooks.com/item/4450593.json

**Reply by 838** [reply]
Hi @mickael-menu 

Thank you for your feedback and time.

Yes, fx. I want to indicate that a publication is available in both audiobook and ePub.

Will this be a valid OPDSPublication? 

```json
{
  "metadata": {
    "@type": "http://schema.org/Book",
    "title": "Last Dance on the Starlight Pier: A Novel"
  },
  "links": [
    {
      "rel": "http://opds-spec.org/acquisition",
      "href": "https://catalog.feedbooks.com/item/4450593/preview",
      "type": "application/epub+zip"
    },
    {
      "rel": "http://opds-spec.org/acquisition",
      "href": "https://catalog.feedbooks.com/item/4450593/preview",
      "type": "application/audiobook+json"
    }
  ]
}
```

**Reply by mickael-menu** [reply]
Yes this looks correct. You can try out your catalog from the Aldiko Next app on iOS or Android, or Thorium from Desktop, to see if it is properly supported.
